common/lib/xmodule/xmodule/course_module.py

```diff
@@ -380,6 +380,15 @@ class CourseFields(object):
         ),
         scope=Scope.settings
     )
+    enable_ccx = Boolean(
+        display_name=_("Enable CCX"),
```

Comment:
You need to add a translators comment to explain what "CCX" means.

http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#python-source-code